tests/unit/classifier/LDA_unittest.cc

```diff
@@ -105,15 +105,15 @@ void test_with_method(
 	generate_test_data<float64_t>(lab, feat);
 
 	auto features = some<CDenseFeatures<float64_t>>(feat);
-	auto labels = some<CBinaryLabels>(lab);
+	auto labels = Some<CLabels>::from_raw(new CBinaryLabels(lab));
```

Comment:
I don't understand why this change is needed. could you explain?

Reply:
ah because of the downcast

Reply:
That's because of `put(std::string, Some<>)`. It works only for types which are `is_sg_base`, therefore I need to pass `CLabels`.

Reply:
yep, all good.